### DISCUSSION TITLE
Building a new Raycast in Electron, fully open

### DISCUSSION THREAD

**Discussion by bm777** [community/community#109460] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I got inspired by the CEO of Perplexity AI. He announced on the X platform that, pplx will be soon in Raycast. Then, I replied, "Really ??? I will do it In 3 hours and post it here [@AravSrinivas](https://twitter.com/AravSrinivas)", then I did it!

And a few seconds later, he shared it in his feed

<img width="604" alt="Screenshot 2024-02-21 at 13 18 20" src="https://github.com/community/community/assets/29865600/b804a217-49f4-4e26-9df9-4d9323b449fc">

 [Tweet](https://x.com/AravSrinivas/status/1759071389099983097)
The project is called [Hask AI](https://github.com/bm777/hask): Quickly search on the internet and receive instant results by just pressing Option+X for Mac or Super+X in Linux/windows

----------
And I super need your feedback
----------
I'm building it and it is open source, and I would like to get your support and contribution.
Roadmap: 
 - integration of other LLM providers: Cohere, OpenAI, Groq, and why not integrating with local LLM inference support (llama.cpp, Ollama, and etc)
 - customization of shortcuts
 - execution of commands and utilities (pallete command from Raycast) -> (in Rust) 
 - cross-platform support
 - and go beyond searching like doing inMemoryRAG

**Comment by Sw333theart** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I got inspired by the CEO of Perplexity AI. He announced on the X platform that, pplx will be soon in Raycast. Then, I replied, "Really ??? I will do it In 3 hours and post it here [@AravSrinivas](https://twitter.com/AravSrinivas)", then I did it!
> 
> And a few seconds later, he shared it in his feed
> 
> <img width="604" alt="Screenshot 2024-02-21 at 13 18 20" src="https://github.com/community/community/assets/29865600/b804a217-49f4-4e26-9df9-4d9323b449fc">
> 
>  [Tweet](https://x.com/AravSrinivas/status/1759071389099983097)
> The project is called [Hask AI](https://github.com/bm777/hask): Quickly search on the internet and receive instant results by just pressing Option+X for Mac or Super+X in Linux/windows
> 
> ----------
> 
> I'm building it and it is open source, and I would like to get your support and contribution.
> Roadmap: 
>  - integration of other LLM providers: Cohere, OpenAI, Groq, and why not integrating with local LLM inference support (llama.cpp, Ollama, and etc)
>  - customization of shortcuts
>  - execution of commands and utilities (pallete command from Raycast) -> (in Rust) 
>  - cross-platform support
>  - and go beyond searching like doing inMemoryRAG

**Reply by hangshan-boop** [reply]
I wish you to make it. Although I don't learn it, it is difficult for you to write code. If you want to do, please conact me.

**Reply by bm777** [reply]
@hangshan-boop i didn't get you!

**Reply by hangshan-boop** [reply]
oh,no. I'm sad.

**Reply by bm777** [reply]
which platform do you have? windows/linux/mac

**Reply by hangshan-boop** [reply]
Of course, Linux

**Reply by hangshan-boop** [reply]
However, Linux that  Fedora, Firefox is bad. Firefox is worse than mobile browser.

**Reply by bm777** [reply]
it is targeted to. I'm building it for Linux too.
By the next week, I will release more distribution support for linux!

**Reply by hangshan-boop** [reply]
you are good. you can tell me, you develop script of progress

**Reply by hangshan-boop** [reply]
I don't be good at script.
